packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1834,10 +1834,11 @@ In case `enum` is used in conjunction with `oneOf`, the set of enumerated values
 
 Okta has also extended [JSON Schema Draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04) with the following keywords:
 
-| Property      | Description                                     | DataType                                                                  | Nullable | Unique | Readonly |
-| :------------- | :----------------------------------------------- | :------------------------------------------------------------------------- | :--------- | :------ | :-------- |
-| required      | determines whether the property is required     | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
-| permissions   | access control permissions for the property     | Array of [Schema property permission](#schema-property-permission-object) | FALSE    | FALSE  | FALSE    |
+| Property                                                        | Description                                                       | DataType                                                                  | Nullable | Unique | Readonly |
+| :-------------------------------------------------------------- | :--------------------------------------------------------------- | :------------------------------------------------------------------------ | :------- | :----- | :------- |
+| required                                                        | Determines whether the property is required                       | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
+| permissions                                                     | Access control permissions for the property                       | Array of [Schema property permission](#schema-property-permission-object) | FALSE    | FALSE  | FALSE    |
+| [unique](#unique-attribute) <ApiLifecycle access="ea" />        | Determines whether the property enforces uniqueness constraint    | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
```

Comment:
"Enforces uniqueness constraint" feels self-referential.  How about: "Determines whether property values must be unique"
(I don't know if it's necessary to include the caveat that empty values don't count.)

Reply:
> I don't know if it's necessary to include the caveat that empty values don't count.

Specified that at L2480.